roles/openshift_facts/library/openshift_facts.py

```diff
@@ -590,13 +518,8 @@ def set_dnsmasq_facts_if_unset(facts):
     """
 
     if 'common' in facts:
-        if 'use_dnsmasq' not in facts['common']:
-            facts['common']['use_dnsmasq'] = bool(safe_get_bool(facts['common']['version_gte_3_2_or_1_2']))
         if 'master' in facts and 'dns_port' not in facts['master']:
-            if safe_get_bool(facts['common']['use_dnsmasq']):
-                facts['master']['dns_port'] = 8053
-            else:
-                facts['master']['dns_port'] = 53
+            facts['master']['dns_port'] = 8053
```

Comment:
At some point I'd like to see this type of stuff bubble up to a default variable in a role.  I don't like digging to figure out information like this.

Reply:
Ok, maybe make a note.  I'm fine putting this into a separate PR.